proposals/evaluation/uc-3-inheritance.md

```diff
@@ -116,29 +119,28 @@ The acl enabling read access to all resources contained in `</weekly-status/>` f
 ```
 
 Any agent making a GET on the `</weekly-status/>` container will receive a `Link: <.acp>; rel="acl"` header in the `401` response that points to the above `<.acl>`. 
-This relation makes the contents of `</weekly-status/.acl>` authoritative and is, therefore, the information the client must use to decide what credentials to present.
+This relation makes the contents of `</weekly-status/.acl>` authoritative and is, therefore, the information the client must use to decide what credentials to present, and the resource's Guard to decide whether to accept those credentials.
+
+At present, though, only the controller of `</weekly-status/.acl>` can read that resource, as WAC does not have an option to make the ACL more widely readable. (Some proposals: adding a [ControlRead](https://github.com/solid/web-access-control-spec/issues/85) mode or [ACLs on ACLs](https://github.com/solid/authorization-panel/issues/189)).
 
-At present, though, only the controller of `</weekly-status/.acl>` can read that resource, as [WAC](https://solid.github.io/web-access-control-spec/#acl-inheritance-algorithm) does not have an option to make the ACL world-readable. 
+#### b. Accessing `</weekly-status/2021-04-28/report.md>`
 
-A unauthenticated client making a GET request to  `</weekly-status/2021-04-28/report.md>` may receive in the header of a `401`, either of the following: 
+A unauthenticated client actor making a GET request to  `</weekly-status/2021-04-28/report.md>` may receive in the header of a `401`, either of the following: 
 
-1) a `Link` to `</weekly-status/.acl>`. 
-2) a `Link` to `</weekly-status/2021-04-28/report.acl>`
+1) `Link: </weekly-status/.acl>; rel=acl`. 
+2) `Link: </weekly-status/2021-04-28/report.acl>; rel=acl`
 
-In the case of (1) the actor should be able to find out what identity to provide by looking at `</weekly-status/.acl>`. 
-But if the controller wants to add a new editor to `<report.md>`, then it will have to add that to the root `</weekly-status/.acl>` as there is no agreed way to create a new acl out of nothing.
+As pointed out previously, only the controller and the resource guard can read those, all other actors (eg. Alice or Bob) need undefined out-of-band knowledge to know how to authenticate. So we are limited to the controller actor.
```

Comment:
```suggestion
As pointed out previously, only the controller and the resource guard can read those; all other actors (e.g., Alice or Bob) need undefined out-of-band knowledge to know how to authenticate. We are therefore limited to the controller actor.
```

Reply:
cheers. I am trying to get Grammarly to work with Visual Studio Code. There is an unofficial plugin, and so it does not work that well.